pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -247,10 +283,25 @@ func validateRuleWithOperations(ruleWithOperations *admissionregistration.RuleWi
 	return allErrors
 }
 
+// hasUnrecognizedVersions returns true of any of the webhooks has an unrecognized version in AdmissionReviewVersions.
+func hasUnrecognizedAdmissionReviewVersions(webhooks []admissionregistration.Webhook) bool {
```

Comment:
function name is misleading, since this can return false for webhooks which do have some unrecognized versions. `hasRecognizedAdmissionReviewVersions` would be clearer, returning true if the existing set of webhooks all had recognized versions.